### ISSUE TITLE
Regularization

### ISSUE THREAD

**Issue by Cre1th** [fizyr/keras-retinanet#219]
Hi, I noticed that keras-retinanet does not use regularization methods like Dropout/BatchNormalization. Is this planned in the future or is it not necessary?

**Comment by hgaiser** [comment]
I'll close this issue, since there's nothing left to do. If you would like to discuss this topic further, my suggestion is to join the Slack channel as is described in the README.
